code/modules/maps/spawners/spawners.dm

```diff
@@ -954,9 +954,7 @@
 	chance = 30
 	amount = 1
 	jiggle = 5
-	to_spawn = list(//obj/item/trash - /obj/item/trash,
-	/obj/item/weapon/shard
-	)
+	to_spawn = pick(existing_typesof(/obj/item/trash))
```

Comment:
The to_spawn list is a list of things it can pick from, so just use to_spawn = existing_typesof(/obj/item/trash)

Reply:
That doesn't compile either believe it or not, it's really determined to only accept a constant. 
(also I didn't mean to push this I just needed to switch branches)

Reply:
ah, turns out existing_typesof is a proc, whereas typesof and subtypesof would be caches perhaps?